composer.json

```diff
@@ -14,7 +14,8 @@
     "require": {
         "php": ">7.2.5",
         "giggsey/libphonenumber-for-php": "^8.0",
-        "symfony/framework-bundle": "^3.4|^4.3|^5.0"
+        "symfony/framework-bundle": "^3.4|^4.3|^5.0",
+        "symfony/intl": "^3.4|^4.3|^5.0"
```

Comment:
Good point, the intl component is no longer embedded in the form component since 5.3 👍 
Just for the record : https://github.com/symfony/symfony/pull/40298